testenrichers/ejb-jakarta/src/main/java/org/jboss/arquillian/testenricher/ejb/EJBInjectionEnricher.java

```diff
@@ -0,0 +1,306 @@
+/*
+ * JBoss, Home of Professional Open Source
+ * Copyright 2009 Red Hat Inc. and/or its affiliates and other contributors
```

Comment:
Not sure if that is much relevant, but shouldn't we bump the year?

```suggestion
 * Copyright 2020 Red Hat Inc. and/or its affiliates and other contributors
```